eng-paths/sr_software_engineer_3.md

```diff
@@ -0,0 +1,381 @@
+- Able to analyze and execute cross-team engineering initiatives by leading the scope of implementation within own team.
+- Communicates clearly with all stakeholders, being able to articulate trade-offs and shift the priorities when necessary.
+- Helps to create buy-in on the decisions (together with the EM and PM) by helping the team to understand the "why" of the decisions being taken.
+
+</td></tr>
+<tr><td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td></tr>
+<tr><td markdown="1" style="font-weight:bold">
+
+- Thrives in communication, ensuring clarity and objectiveness for individuals with varying levels of technical expertise.
+- Able to clearly articulate pros/cons, costs and tradeoffs of possible decisions.
+- Proactively establishes the team’s vision and strategic direction.
+- Inspires and motivates team members towards a shared vision, cultivating a culture of innovation and excellence.
+- Helps on the professional development of team members through mentorship, knowledge sharing, and promoting continuous learning .
+- Holds the team accountable to high technical standards.
+
+</td></tr>
+<tr><td>
+
+Staff Engineer
+
+</td></tr>
+<tr><td  markdown="1">
+
+- Multiplies the effectiveness of others by facilitating cross-team work.
+- Primarily acts as a multiplier by building systems, authoring tools, or introducing policies or patterns that raise the level of productivity of the entire org.
+- Listens and guides debates to help reach decisions, drives commitment and accountability; clearly communicates to and backs all stakeholders.
+- Sets medium-term strategic technical direction, able to look ahead 6 months, identify the areas of greatest need across the engineering org, create plans to improve them, and build teammate and exec buy-in around the plan.
+- Serves as a technical advisor and works closely with senior management.
+- Clearly communicate the status and impact of critical outages and high-impact issues in the appropriate public channels for visibility and awareness.
+- Consistently provides feedback to engineers, with the intention of supporting their career growth and professional development.
+- Leverage communication and expertise by working closely with the teams to ensure that cross-team work is being executed successfully, within the scope and timelines required.
+
+</td></tr>
+</table>
+
+
+### Competencies focus
+<table  markdown="1">
+<tr>
+<td>
+
+Sr Software Engineer 2
+
+</td>
+<td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td>
+<td>
+
+Staff Engineer
+
+</td>
+</tr>
+<tr>
+<td  markdown="1">
+
+- Deliver Results.
+- Service Oriented.
+
+</td>
+<td  markdown="1" style="font-weight:bold">
+
+- Deliver results.
+- Stretegic thinking.
+
+</td>
+<td  markdown="1">
+
+- Deliver results.
+- Stretegic thinking.
```

Comment:
Typo in Strategic

Reply:
lol where did I get that from 🤦‍♀️🤣 thanks!